db/seeds.rb

```diff
@@ -7,3 +7,4 @@
 # it should be safe to run multiple times in the same environment, and to remove
 # the relevant ENV vars after it has run at least once.
 RolesConversion.new.ncr_budget_approvers
+RolesConversion.new.gsa18f_approvers
```

Comment:
I am afraid that adding this is going to make our slow test suite even slower -- did working on this PR give you any good ideas on how we could move away from needing to run seeds before each test? I haven't put much thought into it recently but I am planning on doing so soon!